packages/spaces/src/linkHandlers.js

```diff
@@ -3,45 +3,38 @@ import nativeForm from '@availity/native-form';
 import { isAbsoluteUrl } from '@availity/resolve-url';
 import { updateTopApps, updateUrl } from './helpers';
 
-export const openLink = async (space, { akaname, linkAttributes }) => {
+export const openLink = async (space, { akaname, payerSpaceId }) => {
   if (!space?.link?.url) {
     return;
   }
 
-  await updateTopApps(space.configurationId, space.type, akaname);
+  await updateTopApps(space, akaname);
 
   const target = getTarget(space.link.target);
   const url = !isAbsoluteUrl(space.link.url)
     ? getUrl(
-        updateUrl(
-          space.link.url,
-          'spaceId',
-          space.parents && space.parents[0] ? space.parents[0].id : linkAttributes.spaceId
-        ),
+        updateUrl(space.link.url, 'spaceId', space.parents && space.parents[0] ? space.parents[0].id : payerSpaceId),
```

Comment:
I think we can change the last arg to be a little shorter with
```js
space.parents?.[0]?.id || payerSpaceId
```
what do you think?

Reply:
You so fancy 😍 . I'll update it.